src/actions/InitActions.js

```diff
@@ -0,0 +1,18 @@
+/* jshint node:true, esnext:true */
+
+const c = require("lib/constants");
+const {request, receive} = require("lib/utils");
+const gUserDatabase = require("lib/userDatabase");
+const gBlockedLinks = require("lib/blockedLinks");
+const async = require("lib/async");
+
+module.exports = {
+  initUserDatabase() {
```

Comment:
I think in this case it would be better to make this a `userDatabase` action or something, since `Init` is pretty general and implies a blocking behaviour